charts/scalardl-audit/values.yaml

```diff
@@ -9,6 +9,10 @@ nameOverride: ""
 fullnameOverride: ""
 
 envoy:
+  # -- enable envoy
+  enabled: true
+  # -- String to partially override envoy.fullname template
+  nameOverride: audit
```

Comment:
The output looks good to me but I'm just wondering why it is not consistent with other charts. (the reason is most probably I'm not experienced enough with Helm and k8s)

So, for the scalardb and scalardl chart, `nameOverride` are `scalardb` and `scalardl` respectively.
But, why `nameOverride` is `audit` for `scalardl-audit` chart?
Can it be `scalardl-audit`?

Reply:
I apologize for the inconvenience. :bow:
You can use `scalardl-audit` without any problem.
This is because the name of the release was adjusted to `scalardl-audit`.
 
 Please wait a little longer while I fix it.

Reply:
@feeblefakie 

Please check it out. :bow:
https://github.com/scalar-labs/helm-charts/pull/29/commits/a7ef6fe849a427c9a1753c0ad5f430779e61b6f9


```sh
helm install --generate-name charts/scalardl --namespace default -f ./.conf/scalardl-custom-values.yaml
helm install --generate-name charts/scalardb --namespace default -f ./.conf/scalardb-custom-values.yaml
helm install --generate-name charts/scalardl-audit --namespace default -f ./.conf/scalardl-audit-custom-values.yaml
```
``` sh
helm ls -n default
NAME                            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
scalardb-1632450334             default         1               2021-09-24 11:25:38.086804 +0900 JST    deployed        scalardb-1.0.0          3.1.0
scalardl-1632450294             default         1               2021-09-24 11:24:58.228448 +0900 JST    deployed        scalardl-2.1.0          3.0.1
scalardl-audit-1632450375       default         1               2021-09-24 11:26:19.658497 +0900 JST    deployed        scalardl-audit-1.0.0    3.0.1
```

``` sh
$ kubectl get pods
NAME                                                 READY   STATUS      RESTARTS   AGE
scalardb-1632450334-7988fd95b6-6t95p                 1/1     Running     0          3m5s
scalardb-1632450334-7988fd95b6-cfrxw                 1/1     Running     0          3m5s
scalardb-1632450334-7988fd95b6-vvvq5                 1/1     Running     0          3m5s
scalardb-1632450334-envoy-6d78bc9795-cbhxn           1/1     Running     0          3m5s
scalardb-1632450334-envoy-6d78bc9795-gvlhh           1/1     Running     0          3m5s
scalardb-1632450334-envoy-6d78bc9795-j7shp           1/1     Running     0          3m5s
scalardl-1632450294-envoy-6857559f99-2mdwr           1/1     Running     0          3m46s
scalardl-1632450294-envoy-6857559f99-8mmfs           1/1     Running     0          3m45s
scalardl-1632450294-envoy-6857559f99-nbshq           1/1     Running     0          3m46s
scalardl-1632450294-ledger-58d677d56c-lrr4b          1/1     Running     0          3m46s
scalardl-1632450294-ledger-58d677d56c-nh2hr          1/1     Running     0          3m45s
scalardl-1632450294-ledger-58d677d56c-zhdnz          1/1     Running     0          3m45s
scalardl-audit-1632450375-auditor-77b5ccdd8c-62kjh   1/1     Running     0          2m24s
scalardl-audit-1632450375-auditor-77b5ccdd8c-77gfb   1/1     Running     0          2m24s
scalardl-audit-1632450375-auditor-77b5ccdd8c-lbsmd   1/1     Running     0          2m24s
scalardl-audit-1632450375-envoy-d5b8bb799-bdvtl      1/1     Running     0          2m24s
scalardl-audit-1632450375-envoy-d5b8bb799-kdsz8      1/1     Running     0          2m24s
scalardl-audit-1632450375-envoy-d5b8bb799-r6mfn      1/1     Running     0          2m24s
```

``` sh
$ kubectl get svc
NAME                                      TYPE           CLUSTER-IP       PORT(S)                           AGE
kubernetes                                ClusterIP      172.20.0.1       443/TCP                           55m
scalardb-1632450334-envoy                 LoadBalancer   172.20.104.72    60051:30158/TCP,50052:30762/TCP   3m39s
scalardb-1632450334-envoy-metrics         ClusterIP      172.20.159.146   9001/TCP                          3m39s
scalardb-1632450334-headless              ClusterIP      None             50051/TCP                         3m39s
scalardb-1632450334-metrics               ClusterIP      172.20.21.103    8080/TCP                          3m39s
scalardl-1632450294-envoy                 LoadBalancer   172.20.20.190    50051:32128/TCP,50052:31764/TCP   4m20s
scalardl-1632450294-envoy-metrics         ClusterIP      172.20.113.109   9001/TCP                          4m20s
scalardl-1632450294-headless              ClusterIP      None             50051/TCP,50053/TCP,50052/TCP     4m20s
scalardl-1632450294-metrics               ClusterIP      172.20.127.49    8080/TCP                          4m20s
scalardl-audit-1632450375-envoy           LoadBalancer   172.20.131.126   40051:32259/TCP,40052:30239/TCP   2m58s
scalardl-audit-1632450375-envoy-metrics   ClusterIP      172.20.195.161   9001/TCP                          2m58s
scalardl-audit-1632450375-headless        ClusterIP      None             50051/TCP,50053/TCP,50052/TCP     2m58s
scalardl-audit-1632450375-metrics         ClusterIP      172.20.158.96    8080/TCP                          2m58s
```

``` sh
kubectl get pods scalardl-1632450294-envoy-6857559f99-2mdwr -o json | jq '.spec.containers[0].env[1]'
{
  "name": "scalardl_address",
  "value": "scalardl-1632450294-headless"
}
kubectl get pods scalardb-1632450334-envoy-6d78bc9795-cbhxn -o json | jq '.spec.containers[0].env[1]'
{
  "name": "scalardl_address",
  "value": "scalardb-1632450334-headless"
}
kubectl get pods scalardl-audit-1632450375-envoy-d5b8bb799-bdvtl -o json | jq '.spec.containers[0].env[1]'
{
  "name": "scalardl_address",
  "value": "scalardl-audit-1632450375-headless"
}
```